### ISSUE TITLE
I would like to ask what is the goal of the network trained using Main.py?

### ISSUE THREAD

**Issue by AlanLouis12** [zoubohao/DenoisingDiffusionProbabilityModel-ddpm-#11]
In the Diffusion.py file I see that
loss = F.mse_loss(self.model(x_t, t), x_0, reduction='none')
Does this mean that we want to train the Unet network to make the output images closer and closer to the noise? I don't really understand it here.

**Comment by zoubohao** [comment]
Yes, the model is trained to predict the noise. I recommend you read the paper of DDPM. It is better for you to understand the process of DDPM.
